Cargo.toml

```
@@ -11,6 +11,8 @@ ark-std = "^0.4.0"
 ark-crypto-primitives = { version = "^0.4.0", default-features = false, features = ["r1cs", "sponge"] }
 ark-relations = { version = "^0.4.0", default-features = false }
 ark-r1cs-std = { version = "^0.4.0", default-features = false }
+tiny-keccak = { version = "2.0", features = ["keccak"] }
+sha3 = "0.10.8"
```

```suggestion
sha3 = "0.10"
```

Let's allow the minor to automatically update (although IIRC Cargo did it anyways). But JIC.

---

Also, do we use `sha3` instead of `sha256` for some reason?